### PR TITLE
feat(website): add link to rule source on docs page

### PR DESCRIPTION
**Pull request by DonIsaac**
image

### CODE REVIEW

**Comment by DonIsaac** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4995?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4995** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4995?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4994** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4994?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4990** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4990?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4989** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4989?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @DonIsaac and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: Cargo.toml [hunk 1]
```diff
@@ -200,6 +200,7 @@ oxc_ast_macros.opt-level = 1
 # See: https://insta.rs/docs/quickstart/#optional-faster-runs
 insta.opt-level   = 3
 similar.opt-level = 3
+website.debug     = true
```
Comment (camc314) [inline Cargo.toml:203]: intentional?

File: tasks/website/src/linter/rules/doc_page.rs [hunk 2]
```diff
@@ -49,6 +52,34 @@ pub fn render_rule_docs_page(rule: &RuleTableRow) -> Result<String, fmt::Error>
     }
 
     // TODO: link to rule source
```
Comment (camc314) [inline tasks/website/src/linter/rules/doc_page.rs:54]:
```suggestion
```

**Review by camc314 — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (DonIsaac) [reply Cargo.toml:203]: Oops 👀

**Review by camc314 — APPROVED** [review]

**Pull request merged**